releasenotes/notes/speedup-import-b542f7a8323ef376.yaml

```diff
@@ -1,5 +1,4 @@
 ---
-prelude: >
```

Comment:
The `prelude` section seems empty to me, so I removed it.
If I am wrong and it includes the following section, I can simply rename it.